### PR TITLE
Make clear the tree doesn't precisely represent the spec

### PR DESCRIPTION
**Pull request by hasufell**

### CODE REVIEW

**Comment by hasufell** [comment]
> I wonder whether there is any value in the decision tree, if it does not reflect the policy in full and precisely. 

Maybe, but I don't know if we ought to maintain it.

Seems more like something that could live in hackage docs.

**Comment by gbaz** [comment]
People very much appreciated the decision tree when it was introduced. Many people are better at scanning such things visually than just putting stuff together from the written form. I would be ok with it moving to the faq, since it isn't exactly part of the spec, but it would be a shame to lose it, and very weird and confusing to move it to hackage docs, where it would be distinct from all the other basic pvp reference material.

**Comment by gbaz** [comment]
That said, as it stands, I think the "incomplete" warning suffices and doesn't necessarily need further explanation if that would invoke a new version.

**Comment by hasufell** [comment]
> I would be ok with it moving to the faq

I have the feeling FAQ is getting overloaded with things that aren't necessarily frequently asked questions, but merely informal explanations of the spec.

Maybe this needs restructuring as well.

> and very weird and confusing to move it to hackage docs

Why? Hackage has its own versioning policy and it makes sense to me that it hosts such documentation to aid people who are uploading packages.

**Comment by gbaz** [comment]
Hackage does not have its own versioning policy. It directs people to the PVP website, and therefore it is good to keep all information in one place. This diagram is intended to help explain the pvp, and therefore it makes sense to me that it lives in as close proximity to the pvp as possible.

**Comment by hasufell** [comment]
> Hackage does not have its own versioning policy

It does not enforce PVP and supports the caret operator, which is not part of the spec. 

It has already deviated.

I'm pro keeping the spec lean and making it proper, so I'm fine with removing the decision tree.

**Comment by gbaz** [comment]
This is offtopic, but for clarity, here is how I understand it:

Not mandating the PVP is not a different versioning policy, it is just an upload guidline with regards to the _same_ versioning policy.

The caret operator is not part of hackage, but something supported by cabal files. There is no "alternate" policy which applies to its use, but rather a heuristic which exists in the cabal documentation. It is orthogonal to the PVP, for which purposes, at the moment it is treated purely as syntactic sugar.

**Comment by Bodigrim** [comment]
Fair enough, let's not scope creep and just add a disclaimer. 

For anyone wondering what's imprecise at the moment: the PVP does not say anything about semantical changes, while the decision tree, if understood strictly, mandates a major version bump for any bug fix. We should find a middle ground and reconcile them in the future, but in the meantime it's worth to say that the decision tree is not accurate and serves illustrative purposes only.

Dear CLC members, let's vote on this PR. 
@tomjaguarpaw @hasufell @mixphix @chshersh @angerman @parsonsmatt 

----

+1 from me.

**Comment by tomjaguarpaw** [comment]
+1

File: v1.1/pvp-specification.md [hunk 1]
```diff
@@ -176,7 +176,9 @@ version `1.0.2014` with tags `01` and `27`.
 Decision tree
 -------------
 
-The (incomplete!) decision tree summarises the PVP rules in a concise form
+The (incomplete!) decision tree summarises the PVP rules in a concise form.
+It is not a precise representation of the spec and may contain informal
+conventions and recommendations.
```
Comment (parsonsmatt) [inline v1.1/pvp-specification.md:181]: This is the addition in this PR and I don't see it adding any value. The prior line already calls out "incomplete" and "summarises", both of which point to potential vagueness and infelicity.

I'd rather see this specifically call out how the decision tree differs from the formal spec.

**Comment by mixphix** [comment]
+1

File: v1.1/pvp-specification.md (hunk 1 above)
Comment (Bodigrim) [reply v1.1/pvp-specification.md:181]: IMHO "incomplete" and "summarises" make an impression that the actual PVP demands more, but certainly not less, which is wrong. My experience is that people often skip the specification and refer to the tree as a source of truth, so being a bit louder in the disclaimer might help.

I absolutely agree that this area needs more future work. I'd suggest not to scope creep now though.

**Comment by hasufell** [comment]
+1

**Comment by angerman** [comment]
+1

**Comment by chshersh** [comment]
+1

**Comment by Bodigrim** [comment]
Thanks all, that's enough votes to approve.

**Pull request merged**